setup.py

```diff
@@ -26,7 +26,7 @@
     author="Anders Huss",
     author_email="andhus@kth.se",
     license='MIT',
-    install_requires=['scantree>=0.0.1'],
+    install_requires=['scantree>=0.0.2', 'pathspec<=0.10.0'],
```

Comment:
In fact `pathspec==0.10.0` was "yanked" of pypi https://pypi.org/project/pathspec/0.10.0/ so this should really be `<0.10.0` to be clear. Will fix in #17